definitions/pipelines/bisulfite.cwl

```diff
@@ -30,6 +30,10 @@ inputs:
         type: int
     QCannotation:
         type: File
+#nonmethylation_sites_bed is a variable to determine if user wants to obtain vcf/bed/bedgraph/bigwig for non - methylation sites. Value has to be yes or no
+    assay_non_cpg_sites:
+        type: string?
+        default: 'no'
```

Comment:
Did you explore whether a boolean would work here?  What happens when someone puts in "false" in the input yaml?  (but then ends up getting all the non-cpg sites anyway, as written...)

Reply:
If not converted to a boolean, then it would be good to at least constrain the values using an enum.  For an example:
https://github.com/genome/analysis-workflows/blob/896dd42957d4a2c2d275420b5b1f1ac09b8f5c82/definitions/tools/deeptools_bamcoverage.cwl#L20-L23

Reply:
Changed it to enum.